code/modules/cargo/exports/food_and_drink.dm

```diff
@@ -16,3 +16,46 @@
 	if(sold_food.silver_spawned)
 		return FOOD_WORTHLESS
 	return sold_food.value
+
+// Humanlike meats
+/datum/export/meat/human
```

Comment:
All of these should use the value var on their actual food/snacks var, look at existing food items to see what I mean.

Reply:
I know that but that doesn't let me put them in EXPORT_CONTRABAND

Reply:
`export_category = EXPORT_CONTRABAND` is all that needs to be set for humanoid meat to be considered contraband, EXPORT_CONTRABAND is for foods that are meant to be scaled as very expensive or difficult to export (Like Energy Cake.)